static/app/components/searchQueryBuilder/hooks/useQueryBuilderState.tsx

```diff
@@ -656,18 +658,126 @@ function updateFilterKey(
   };
 }
 
+function isReplaceToken(
```

Comment:
nit: could use a more descriptive name or a comment explaining what this is useful for